examples/pipeline/gitlab-ci.yml

```diff
@@ -27,6 +27,9 @@ include: ${CI_PROJECT_DIR}/.gitlab/ci/login.yml
 
 # build management-container
 build-dmc:
+  services:
```

Comment:
in our internal gitlab runner infrastruture not needed, because this is auto included

Reply:
Yes, but not everybody is using our internal infrastructure. :)

Reply:
I agree with you.